modules/metering-troubleshooting.adoc

```diff
@@ -10,10 +10,173 @@ A common issue with metering is pods failing to start. Pods might fail to start
 [id="metering-not-enough-compute-resources_{context}"]
 == Not enough compute resources
 
-A common issue when installing or running metering is lack of compute resources. Ensure that metering is allocated the minimum resource requirements described in the installation prerequisites.
+A common issue when installing or running metering is a lack of compute resources. As the cluster grows and more reports are created, the `reporting-operator` pod requires more memory. If memory usage reaches the pod limit, the cluster terminates the pod with an `OOMKilled` status. Ensure that metering is allocated the minimum resource requirements described in the installation prerequisites.
```

Comment:
Dunno if worth it, but wondering if there's a good way to sneak in the definition of "OOM" here in this first related instance (didn't see it mentioned explicitly in IBM SG or RH supplemental guide). Maybe something like:

> If memory usage reaches the pod limit, the cluster terminates the pod with an `OOMKilled` status due to an out of memory (OOM) event.

or

> If memory usage reaches the pod limit, the cluster considers the pod out of memory (OOM) and terminates it with an `OOMKilled` status.

Then the later, non-marked-up instances of the "OOM" acronym will be covered.

Reply:
Great suggestion, thank you.